candi/openapi/doc-ru-node_group.yaml

```diff
@@ -72,6 +72,8 @@ spec:
                   description: "The summary number of overprovisioned Nodes for this NodeGroup in all zones."
                 error:
                   description: "Error message about possible problems with the group handling."
+                kubernetesVersion:
+                  description: "The current version of kubernetes on the nodes, or the version to which the nodes will be upgraded."
```

Comment:
doc-ru-* should have RU description :)

Reply:
All statuses written earlier were in English. Should I rewrite statuses descriptions to the Russian language for the rest?

Reply:
@alex123012 Please delete the status section from the doc-ru-*.yaml. The status section of a CRD is not rendered on the side. Thanks

Reply:
Or simply don't add the kubernetesVersion parameter and live it as is.